manifest.json.template

```diff
@@ -106,6 +106,11 @@
       "artifact": "apiml-sample-extension-*.zip",
       "exclusions": ["*PR*.zip"]
     },
+    "org.zowe.apiml.cloud-gateway-package": {
```

Comment:
If we add a new component, think we also need to increase this number https://github.com/zowe/zowe-install-packaging/blob/v2.x/master/.github/workflows/build-packaging.yml#L174.

From this build log https://github.com/zowe/zowe-install-packaging/runs/8266699986?check_suite_focus=true, seems cloud-gateway is not downloaded. This could be caused by the version definition in manifest.json.template is not correct.

Reply:
Thanks for noticing it Jack. I've updated the variable. 
About the artefact version, you're right, I will change it to the correct one